lib/index.js

```diff
@@ -147,7 +147,7 @@ AxeBuilder.prototype.analyze = function(callback) {
       axeSource: source,
       config,
       options: this._builderOptions
-    });
```

Comment:
uhhh

Reply:
?

Reply:
Formatting (looks like two tabs worth of indentation was added)